Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/service/impl/ConceptMergeService.java

```diff
@@ -0,0 +1,297 @@
+package edu.asu.conceptpower.app.service.impl;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import edu.asu.conceptpower.app.bean.ConceptsMergeBean;
+import edu.asu.conceptpower.app.core.ConceptTypesService;
+import edu.asu.conceptpower.app.core.ConceptTypesService.ConceptTypes;
+import edu.asu.conceptpower.app.core.IConceptListManager;
+import edu.asu.conceptpower.app.core.IConceptManager;
+import edu.asu.conceptpower.app.core.IConceptTypeManger;
+import edu.asu.conceptpower.app.exceptions.DictionaryDoesNotExistException;
+import edu.asu.conceptpower.app.exceptions.DictionaryModifyException;
+import edu.asu.conceptpower.app.exceptions.IndexerRunningException;
+import edu.asu.conceptpower.app.exceptions.LuceneException;
+import edu.asu.conceptpower.app.service.IConceptMergeService;
+import edu.asu.conceptpower.core.ConceptEntry;
+import edu.asu.conceptpower.core.ConceptList;
+import edu.asu.conceptpower.core.ConceptType;
+
+@Service
+public class ConceptMergeService implements IConceptMergeService {
+
+    @Autowired
+    private IConceptTypeManger conceptTypeManager;
+
+    @Autowired
+    private IConceptManager conceptManager;
+
+    @Autowired
+    private IConceptListManager conceptListManager;
+
+    @Autowired
+    private ConceptTypesService conceptTypesService;
+
+    @Override
+    public ConceptsMergeBean prepareMergeConcepts(List<ConceptEntry> conceptEntries,
+            ConceptsMergeBean conceptsMergeBean) {
+        for (ConceptEntry entry : conceptEntries) {
+
+            if (!isNullOrEmpty(entry.getPos())) {
+                conceptsMergeBean.setSelectedPosValue(entry.getPos().toLowerCase());
+            }
```

Comment:
Instead of all this ```isNullOrEmpty``` method callsed, why don't you just check for null/empty when adding to the new set?